### ISSUE TITLE
BUG - sev-3 - Android - Rx - "Select all" checkbox of multi-med refill read incorrectly by screen reader

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#4755]
## All of the writeup below is directly copy-pasted from #4273
The bug described in that ticket (the nearly-permanent insertion of the word 'mixed' into the select all checkbox screen reader text) was unfortunately never addressed/fixed by the PR attached to that ticket. (The PR attached to that ticket added the callout "select all" to the checkbox, which is nice, but doesn't fix the original bug writeup)

<br>
<br>
<br>

## What happened?

The multi-med refill header aren't being read correctly by Android TalkBack.

"Select all [ ]" is being read correctly the first time, but then if you select a med (aka get it into a 'mixed' state), it will subsequently always read "mixed" instead of the label, even if you have everything selected, or everything not selected. [Check out the video](https://drive.google.com/file/d/14_k6ypYnctYttm7lu7m3VmPYUga6iXdV/view?usp=share_link). <- no sound, but you can see the focus and the captions for what's being read out.

I'm guessing that this was introduced by #4038 since it appears that the 'mixed' state is fouling things up

## Specs:

- Device: Samsung tablet and Google Pixel
- OS: Android
- User Account: karsyn (meds testing user)
- Accessibility State:

## Steps to Reproduce

1. Go to prescriptions > "request refills" (button at bottom of list/screen)
2. Swipe down to interact with the headers, as per the video above

## Desired behavior

The correct status of the checkbox (not checked, mixed, checked) should be read, no matter if the user has gotten into a 'mixed' state previously or not

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Issue Tracking](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/testing/VA%20Mobile%20App%20Test%20Plan.md#issue-tracking) for details on Severity Levels

- 3 - Low

## Linked to Story

Found during testing for #4263 

## Ticket Checklist
- [ ] Steps to reproduce are defined
- [ ] Desired behavior is added
- [ ] Labels added (front-end, back-end, feature, bug)
- [ ] Linked to an Epic (General Tech/Design Debt & Maintenance)

**Comment by timwright12** [comment]
@dumathane @bischoffa thoughts about getting this worked in? It was originally filed in February 2023. We've since upgraded React Native, so maybe something there? Even if we can't fix this it would be good to document and close out.

**Comment by bischoffa** [comment]
@timwright12 we can look into next sprint. I prioritized other bugs over this one cause the 5 points doesn't guarantee a fix because of the previous review and documentation. Also I thought this has some tie in with [4145](https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/4145) which hasn't been prioritized since 10/2022 open date.

**Comment by timwright12** [comment]
Yeah good point on #4145. Do you think we need some kind of alert for super old issue just to ping us so we don't forget them? I'm sure something like that could be automated

**Comment by bischoffa** [comment]
I would be open to that idea! Until then I would recommend that Engineering regularly reviews their specific epics and backlogs.  This may also give engineering ideas on how to spend excess capacity. 

I review the bug epic at least 1x a month and see if blocked / dependent tickets can be worked. I occasionally post comments on old tickets for engineering updates.

**Comment by timwright12** [comment]
Sounds good, I'll put some thought into it. Thanks!

**Comment by Sparowhawk** [comment]
@rbontrager when you have some time can you confirm this one is still an issue? We updated RN a few times and also updated our Android requirement since this bug was reported and I believe it may have been fixed with the update.

**Comment by alexandec** [comment]
I can confirm that the issue of an incorrect "mixed" announcement is still present as of May 2024 in my Android simulator with TalkBack on. I tried supplying the 'aria-checked' prop but that had no effect. I also tried disabling and re-enabling the checkbox but that didn't fix the issue either.

**Comment by bischoffa** [comment]
Unassigned Tom as he is not the QA for Health and assigned Rachael. 

Rachael I know there is a question above for your but I don't see this being tagged to a sprint as of yet.

**Comment by Sparowhawk** [comment]
I have confirmed this is a RN bug. I created an issue on the RN github repo so they can fix it

https://github.com/facebook/react-native/issues/44960

**Comment by TKDickson** [comment]
Previously we didn't have official guidance on when to close tickets as not fixing, but we do now.

Externally-owned bugs (ones we can't fix because we don't own the code) should be closed after reporting to the responsible team (Dylan's comment above). So closing now.
